gradle.properties

```diff
@@ -97,38 +97,32 @@ kotlin.mpp.import.enableKgpDependencyResolution=true
 jetbrains.compose.compiler.version=1.5.14.1
 
 # Compose Multiplatform doesn't have its own implementation for Android target.
-# When user adds a dependency on `org.jetbrains.compose.*` in Android target, in fact `android.compose` is used.
-# To know which version, should be used, see compose/frameworks/support/libraryversions.toml
-artifactRedirecting.publication=true
-# Look for `COMPOSE` in libraryversions.toml
-artifactRedirecting.androidx.compose.version=1.8.0-beta02
-# Look for `COMPOSE_MATERIAL3` in libraryversions.toml
-artifactRedirecting.androidx.compose.material3.version=1.3.1
-artifactRedirecting.androidx.compose.foundation.version=1.8.0-beta02
-artifactRedirecting.androidx.compose.material.version=1.8.0-beta02
-# Look for `COMPOSE_MATERIAL3_COMMON` in libraryversions.toml
-artifactRedirecting.androidx.compose.material3.common.version=1.0.0-alpha01
-# Look for `COMPOSE_MATERIAL3_ADAPTIVE` in libraryversions.toml
-artifactRedirecting.androidx.compose.material3.adaptive.version=1.1.0-beta01
-# We use artifactRedirecting not only for Compose libs:
-artifactRedirecting.androidx.collection.version=1.5.0
-artifactRedirecting.androidx.annotation.version=1.9.1
-artifactRedirecting.androidx.graphics.version=1.0.1
-artifactRedirecting.androidx.lifecycle.version=2.9.0-alpha12
-artifactRedirecting.androidx.navigation.version=2.9.0-alpha08
-artifactRedirecting.androidx.performance.version=1.0.0-alpha01
-artifactRedirecting.androidx.savedstate.version=1.3.0-alpha10
-artifactRedirecting.androidx.window.version=1.4.0-alpha04
+# When user adds a dependency on `org.jetbrains.compose.*` in Android target,
+# in fact `android.compose` is used.
+#
+# Versions should equal the last merged to the branch
+artifactRedirection.version.androidx.compose=1.8.0-beta02
```

Comment:
Moved "version" from the end, to keep the dynamic part (`androidx.compose`) not in the middle